dev_test_cmd.sh

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
```

Comment:
I usually like to `set -eux pipefail` for better bash safety. Pipefail is pretty optional since it can cause some unexpected / unwanted results.

What's the purpose of using `env` here?

Reply:
Mostly, it's muscle memory for increased portability: `#!/usr/bin/env` searches PATH for `bash`, since `bash` is not always in `/bin`, particularly on non-Linux (but still POSIX compliant) systems. For example, on some BSD systems, it's in `/usr/local/bin`, since it was installed as an optional package. However, under further consideration I suppose this could pose a security risk (e.g. a malicious binary named `bash` exported into the path).

Reply:
I don't think it's a security risk - if someone has a malicious binary in their $PATH they've already been compromised in some way. It's just different than how we've done bash scripts so far and I wanted to understand it. I've seen the env trick for Ruby and Python before, but not for bash. Super interesting to know that bash is installed in different places on BSD systems!

Reply:
settled on `set -eu` these are good callouts, thank you

> I usually like to `set -eux pipefail` for better bash safety. Pipefail is pretty optional since it can cause some unexpected / unwanted results.

settled on `set -eu` these are good callouts, thank you